exercises/largest-series-product/example/Series.ceylon

```diff
@@ -0,0 +1,13 @@
+class Error() {}
```

Comment:
any standard `Either`-like type that is recommended for use? Don't see it in https://ceylon-lang.org/blog/2015/12/14/failure/, so just used this. Students are free to add whatever they want to the `Error`.